README.md

```diff
@@ -69,17 +72,15 @@ fastify.get("/sendmail", async (req, reply) => {
 
 #### Templates
 Each message must have the following templates with the file extension set in point-of-view config:
-  - `from`: Contains email address the email is to be sent from.
-  - `subject`: Contains the subject for the email.
-  - `html`: Contains the html for the email.
+  - `html`: Contains the html template for the email.
```

Comment:
i missed these backticks the first time around, let's match this with the new format down below